shared/message/platform/NUSenseData.proto

```diff
@@ -62,6 +62,14 @@ message Servo {
     float voltage = 12;
     /// The last read temperature of the servo
     float temperature = 13;
+    /// The number of successes.
+    uint32 num_successes = 14;
+    /// The number of timeouts.
+    uint32 num_timeouts = 15;
+    /// The number of CRC-errors.
+    uint32 num_crc_errors = 16;
+    /// The number of packet-errors.
+    uint32 num_errors = 17;
```

Comment:
Might be a good idea to wrap these four fields in a submessage, they're all closely related and doing one task together

Reply:
Will act on these suggestions soon.